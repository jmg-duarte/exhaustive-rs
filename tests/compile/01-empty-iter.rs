use exhaustive::c;

fn main() {
    c![];
}