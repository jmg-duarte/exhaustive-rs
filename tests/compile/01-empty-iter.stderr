error: comprehension must not be empty
 --> $DIR/01-empty-iter.rs:4:5
  |
4 |     c![];
  |     ^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
